libs/experimental/langchain_experimental/graph_transformers/llm.py

```diff
@@ -771,6 +771,7 @@ def __init__(
         relationship_properties: Union[bool, List[str]] = False,
         ignore_tool_usage: bool = False,
         additional_instructions: str = "",
+        method: str = "json_schema",
```

Comment:
```suggestion
        method: Literal["function_calling", "json_schema", "json_mode"] = "json_schema",
```
I think these are the 3 methods that would be supported by ollama